lib/ephemeral/client.rb

```diff
@@ -32,6 +38,13 @@ def post(resource, options)
       content
     end
 
+    def get(resource, options)
```

Comment:
change this to `def get(resource, options = {})` so that there is a default option. That way you can just call `get(resource)` without having to add the `{}`.